server/models/Document.js

```diff
@@ -208,11 +208,24 @@ Document.associate = (models) => {
       { model: models.User, as: "updatedBy", paranoid: false },
     ],
   });
-  Document.addScope("withViews", (userId) => ({
-    include: [
-      { model: models.View, as: "views", where: { userId }, required: false },
-    ],
-  }));
+  Document.addScope("withViews", (userId) => {
+    if (!userId)
+      return {
+        include: [
+          {
+            model: models.View,
+            as: "views",
+            required: false,
+          },
+        ],
+      };
```

Comment:
Intrigued by this block? I'm concerned that not passing userId could result in accidentally exposing/loading view data for all users in a team.

Reply:
I add it like this, because i didn't got a way of not loading anything when we don't have a userId.
If you could help me..